walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -367,7 +367,7 @@ public void onlyOptionalAttendeesWithGaps() {
             Arrays.asList(PERSON_A)),
         new Event("Event 3", TimeRange.fromStartEnd(TIME_0830AM, TIME_0930AM, false),
             Arrays.asList(PERSON_B)),
-        new Event("Event 4", TimeRange.fromStartEnd(TIME_1100AM, TimeRange.END_OF_DAY, false),
+        new Event("Event 4", TimeRange.fromStartEnd(TIME_1100AM, TimeRange.END_OF_DAY, true),
```

Comment:
nit: optional - consider adding inline Java comment on primitive type parameters for clarity:

```
TimeRange.fromStartEnd(TIME_1100AM, TimeRange.END_OF_DAY, /* inclusive= */ false)
```

Same comment to the rest of the calls on TimeRange util in this class.